Description: Remove jQuery script tags
 The referenced files are no longer included in the upstream source.
Author: Hugh McMaster
Forwarded: https://gitlab.freedesktop.org/freetype/freetype-web/-/merge_requests/7
Last-Update: 2021-12-28

--- a/ft2docs/docs/glyphs/glyphs-1.html
+++ b/ft2docs/docs/glyphs/glyphs-1.html
@@ -24,12 +24,6 @@
         href="../css/freetype2_-90.css">
 
   <script type="text/javascript"
-          src="../../../js/jquery-1.11.0.min.js">
-  </script>
-  <script type="text/javascript"
-          src="../../../js/jquery.ba-resize.min.js">
-  </script>
-  <script type="text/javascript"
           src="../../../js/freetype2.js">
   </script>
 
--- a/ft2docs/docs/index.html
+++ b/ft2docs/docs/index.html
@@ -22,12 +22,6 @@
         href="css/freetype2_-60.css">
 
   <script type="text/javascript"
-          src="../../js/jquery-1.11.0.min.js">
-  </script>
-  <script type="text/javascript"
-          src="../../js/jquery.ba-resize.min.js">
-  </script>
-  <script type="text/javascript"
           src="../../js/freetype2.js">
   </script>
 
